src/RuleSet/Sets/PERCS1x0Set.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * PER Coding Style v1.0.
+ *
+ * @see https://github.com/php-fig/per-coding-style/blob/1.0.0/spec.md
+ */
+final class PERCS1x0Set extends AbstractRuleSetDescription
+{
+    public function getName(): string
+    {
+        return '@PER-CS1.0';
+    }
+
+    public function getRules(): array
+    {
+        return [
+            '@PSR12' => true,
+        ];
+    }
+
+    public function getDescription(): string
+    {
+        return 'Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.';
```

Comment:
Is the `_` expected to be here?

Reply:
It was in the PERSet class already, so I just kept it: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/RuleSet/Sets/PERSet.php#L35

Whether that was correct in the first place, I cannot say.  I just followed the existing pattern.

Reply:
Sorry for the fuss, apparently that how links work in RST files :)